src/FixedRateIrm.sol

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+import {IIrm} from "../lib/morpho-blue/src/interfaces/IIrm.sol";
+import {IFixedRateIrm} from "./interfaces/IFixedRateIrm.sol";
+
+import {ErrorsLib} from "./libraries/ErrorsLib.sol";
+import {MarketParamsLib} from "../lib/morpho-blue/src/libraries/MarketParamsLib.sol";
+import {Id, MarketParams, Market} from "../lib/morpho-blue/src/interfaces/IMorpho.sol";
+
+/// @title FixedRateIrm
+/// @author Morpho Labs
+/// @custom:contact security@morpho.org
+contract FixedRateIrm is IFixedRateIrm {
+    using MarketParamsLib for MarketParams;
+
+    /* EVENTS */
+
+    /// @notice Emitted when a borrow rate is set.
+    event SetBorrowRate(Id indexed id, uint256 newBorrowRate);
+
+    /* STORAGE */
+
+    /// @notice Borrow rates.
+    mapping(Id => uint256) public _borrowRate;
+
+    /* SETTER */
+
+    /// @inheritdoc IFixedRateIrm
+    function setBorrowRate(Id id, uint256 newBorrowRate) external {
```

Comment:
The argument of `setBorrowRate` is `Id` whereas it's `MarketParams` for `borrowRate*`. Maybe the inconsistency is on purpose to reduce calldata cost but I'm pointing it out just to make sure it's on purpose.

Reply:
both are ok for me, you would prefer consistency?